pkgs/by-name/id/id3/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "id3";
+  version = "0.81";
+
+  src = fetchFromGitHub {
+    owner = "squell";
+    repo = "id3";
+    rev = "${version}";
+    sha256 = "+h1wwgTB7CpbjyUAK+9BNRhmy83D+1I+cZ70E1m3ENk=";
+  };
+
```

Comment:
```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ libiconv ];
```

Reply:
I just checked the ofborg output

Reply:
Ok. I didn't know getting that kind of information from ofborg was possible. Thx :slightly_smiling_face: